scripts/gen_merkle_dist.js

```diff
@@ -1,56 +1,119 @@
 // Script that generates a new Merkle Distribution and outputs the data to JSON files
 
 const fs = require("fs")
+const shell = require("shelljs")
+const dotenv = require("dotenv").config()
 const Subgraph = require("../src/stakingrewards/subgraph.js")
 const Rewards = require("../src/stakingrewards/rewards.js")
 const MerkleDist = require("../src/stakingrewards/merkle_dist.js")
 
+const bonusWeight = 0.0
+const ongoingWeight = 0.875
+const tbtcv2Weight = 0.125
+const startTime = 1664496000 // Sep 30th 2022 00:00:00 GMT
+const endTime = 1667260800 // Nov 1st 2022 00:00:00 GMT
+const lastDistribution = "2022-09-30"
+
+const tbtcv2ScriptPath = "src/tbtcv2-rewards/"
 const graphqlApi =
   "https://api.studio.thegraph.com/query/24143/main-threshold-subgraph/0.0.7"
-const bonusWeight = 1.0 // Stakes receive the full bonus
-const ongoingWeight = 1.0 // Stakes receive the full ongoing rewards
-const startTime = 1654041600 // Jun 1st 2022 00:00:00 GMT
-const endTime = 1664496000 // Sep 30th 2022 00:00:00 GMT
-const endTimeDate = new Date(endTime * 1000).toISOString().slice(0, 10)
-const distribution_path = "distributions/" + endTimeDate
 
 async function main() {
+  let earnedBonusRewards = {}
+  let earnedOngoingRewards = {}
+  let earnedTbtcv2Rewards = {}
+  let bonusRewards = {}
+  let ongoingRewards = {}
+  let tbtcv2Rewards = {}
+  const startDate = new Date(startTime * 1000).toISOString().slice(0, 10)
+  const endDate = new Date(endTime * 1000).toISOString().slice(0, 10)
+  const distPath = `distributions/${endDate}`
+  const lastDistPath = `distributions/${lastDistribution}`
+  const tbtcv2Script =
+  `./rewards.sh ` +
+  `--rewards-start-date ${startDate} ` +
+  `--rewards-end-date ${endDate} ` +
+  `--rewards-json ./rewards.json ` +
```

Comment:
It's no harm adding it explicitly, but this path and name are set by default. We can leave it as is, just FYI.

Reply:
Done!